tests/testthat/test-as_html.R

```diff
@@ -36,6 +36,28 @@ test_that("code inside Sexpr is evaluated", {
   expect_equal(rd2html("\\Sexpr{1 + 2}"), "3")
 })
 
+test_that("can convert cross links to online documentation url", {
+  expect_equal(
+    rd2html("\\link[base]{library}"),
+    link_remote(label = "library", topic = "library", package = "base")
+  )
+})
+
+test_that("can parse local links with topic!=label", {
+  # make a fake index that we can use to search for local help topics as if
+  # we were running build_site on the pkgdown package
+  pkgdownindex = list(name = "build_site",
```

Comment:
Can you fix the indenting here please? (Should be the same as `expect_equal()`).  You can also remove the comments as I don't think they're necessary